02-express-tutorial/app.js

```diff
@@ -1 +1,76 @@
-console.log('Express Tutorial')
+
+
+const express = require('express');
+const app = express();
+
+// Middleware to serve static files from the 'public' directory
+app.use(express.static("./public"));
+
+// Middleware to parse JSON bodies
+app.use(express.json());
+
+const { products } = require("./data");
+
+
+// API endpoint that returns JSON
+app.get('/api/v1/test', (req, res) => {
+    res.json({ message: "It worked!" });
+  });
+
+// API endpoint that returns products JSON
+app.get('/api/v1/products', (req, res) => {
+    res.json(products);
+  });
+
+// API endpoint that returns product JSON by ID
+app.get('/api/v1/products/:productID', (req, res) => {
+    const idToFind = parseInt(req.params.productID); 
+    const product = products.find((p) => p.id === idToFind);
+    if (product)
+    res.json(product);
+    else
+    res.json({ message: "Product Not Found!" });
```

Comment:
Add an indent level to the content of the `if` and `else` blocks.
```suggestion
    if (product)
        res.json(product);
    else
        res.json({ message: "Product Not Found!" });
```